docs/porting-a-v2-ea-to-v3.md

```diff
@@ -120,45 +115,63 @@ const restTransport = new RestTransport<EndpointTypes>({
 })
 ```
 
-A batch warming transport is not much different. Instead of receiving a single `req` in `prepareRequest()` and
-`parseResponse()`, you now receive an array of requests in `params`:
+An example to a batch endpoint is not much different:
 
 ```typescript
-const batchTransport = new BatchWarmingTransport<EndpointTypes>({
+const httpTransport = new HttpTransport<EndpointTypes>({
   prepareRequest: (params, config) => {
     // The `params` param contains an array of the request made to the EA.
     // Using this, return the request config to the DP.
     return {
-      baseURL: config.API_ENDPOINT || DEFAULT_API_ENDPOINT,
-      url: '/price',
-      method: 'GET',
-      params: {
-        from: params.map((req) => req.base).join(','),
-        to: params.map((req) => req.quote).join(','),
-      },
+      params,
+      request: {
+        baseURL: config.API_ENDPOINT || DEFAULT_API_ENDPOINT,
+        url: '/price',
+        method: 'GET',
+        params: {
+          from: params.map((req) => req.base).join(','),
+          to: params.map((req) => req.quote).join(','),
+        },
+      }
     }
   },
   parseResponse: (params, res) => {
     // The `params` param contains an array of the request made to the EA.
     // The `res` param contains the response from the DP.
     // Using this, return an array of each combination request-response combination:
-    // return [
-    //   {
-    //     params: { base: "ETH", quote: "USD" }
-    //     value: 123.45
-    //   },
-    //   {
-    //     params: { base: "BTC", quote: "USD" }
-    //     value: 123.45
-    //   }
-    // ]
+    return [
+      {
+        params: { base: "ETH", quote: "USD" }
+        response: {
+          result: data.price,
```

Comment:
We didn't really do it better before but is it alright that this is a little ambiguous? Not a blocker to get this PR in but maybe we can make this example more realistic with request-response matching? Maybe we can just copy something in from coingecko.

Reply:
Could you expand on this?

Reply:
Sure. It's just that the prepareRequest example is a realistic and looks like something that's used in an EA. But prepareResponse is sort of hard-coded response instead of an example. This isn't necessarily in the scope of the ticket though.

Reply:
Maybe something like this? Unless that's less helpful to visualize the method.
```parseResponse: (params, res) => {
      return params.map((req) => ({
        params: req,
        response: {
          data: res.data,
          statusCode: 200,
          result: res.data.price,
        },
      }))
    },```